src/main/resources/config/liquibase/changelog/20230713171500_changelog.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.18.xsd">
+    <changeSet author="pal03377" id="20230713171500-3">
```

Comment:
almost all our changelogs, do not have the -number in the end. for the id. Is this needed here?

Reply:
This was an artifact from when I generated the changelog. I've changed it 👍